From 2151b7e9098e5c0e342adcc145da6b3f431c1d04 Mon Sep 17 00:00:00 2001
From: wusel <wusel+src@uu.org>
Date: Wed, 24 Jun 2026 23:25:08 +0200
Subject: [PATCH] And more tiny ones ...

---
 targets/ath79-generic | 22 ++++++++++++++++++----
 1 file changed, 18 insertions(+), 4 deletions(-)

diff --git a/targets/ath79-generic b/targets/ath79-generic
index 407dffe4..f9f6aff6 100644
--- a/targets/ath79-generic
+++ b/targets/ath79-generic
@@ -452,11 +452,13 @@ device('tp-link-cpe210-v1', 'tplink_cpe210-v1', {
 		'tp-link-cpe210-v1.0', -- Upgrade from OpenWrt 19.07
 		'tp-link-cpe210-v1.1', -- Upgrade from OpenWrt 19.07
 	},
+	class = 'tiny',
 })
 device('tp-link-cpe210-v2', 'tplink_cpe210-v2', {
 	manifest_aliases = {
 		'tp-link-cpe210-v2.0', -- Upgrade from OpenWrt 19.07
 	},
+	class = 'tiny',
 })
 device('tp-link-cpe210-v3', 'tplink_cpe210-v3', {
 	manifest_aliases = {
@@ -464,16 +466,24 @@ device('tp-link-cpe210-v3', 'tplink_cpe210-v3', {
 		'tp-link-cpe210-v3.1', -- Upgrade from OpenWrt 19.07
 		'tp-link-cpe210-v3.20', -- Upgrade from OpenWrt 19.07
 	},
+	class = 'tiny',
+})
+device('tp-link-cpe220-v3', 'tplink_cpe220-v3', {
+	class = 'tiny',
 })
-device('tp-link-cpe220-v3', 'tplink_cpe220-v3')
 device('tp-link-cpe510-v1', 'tplink_cpe510-v1', {
 	manifest_aliases = {
 		'tp-link-cpe510-v1.0', -- upgrade from OpenWrt 19.07
 		'tp-link-cpe510-v1.1', -- upgrade from OpenWrt 19.07
 	},
+	class = 'tiny',
+})
+device('tp-link-cpe510-v2', 'tplink_cpe510-v2', {
+	class = 'tiny',
+})
+device('tp-link-cpe510-v3', 'tplink_cpe510-v3', {
+	class = 'tiny',
 })
-device('tp-link-cpe510-v2', 'tplink_cpe510-v2')
-device('tp-link-cpe510-v3', 'tplink_cpe510-v3')
 
 device('tp-link-cpe710-v1', 'tplink_cpe710-v1')
 
@@ -537,14 +547,18 @@ device('tp-link-wbs210-v1', 'tplink_wbs210-v1', {
 	manifest_aliases = {
 		'tp-link-wbs210-v1.20', -- upgrade from OpenWrt 19.07
 	},
+	class = 'tiny',
 })
 
-device('tp-link-wbs210-v2', 'tplink_wbs210-v2')
+device('tp-link-wbs210-v2', 'tplink_wbs210-v2', {
+	class = 'tiny',
+})
 
 device('tp-link-wbs510-v1', 'tplink_wbs510-v1', {
 	manifest_aliases = {
 		'tp-link-wbs510-v1.20', -- upgrade from OpenWrt 19.07
 	},
+	class = 'tiny',
 })
 
 -- Ubiquiti
-- 
2.30.2

